ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/CrosslinkCommitteeUtil.java

```diff
@@ -118,12 +153,12 @@ public static Integer compute_shuffled_index(int index, int index_count, Bytes32
       BeaconState state, UnsignedLong slot, UnsignedLong index) {
     UnsignedLong epoch = compute_epoch_at_slot(slot);
     UnsignedLong committees_per_slot = get_committee_count_at_slot(state, slot);
-    final int committeeIndex =
+    int committeeIndex =
         slot.mod(UnsignedLong.valueOf(SLOTS_PER_EPOCH))
             .times(committees_per_slot)
             .plus(index)
             .intValue();
-    final int count = committees_per_slot.times(UnsignedLong.valueOf(SLOTS_PER_EPOCH)).intValue();
+    int count = committees_per_slot.times(UnsignedLong.valueOf(SLOTS_PER_EPOCH)).intValue();
```

Comment:
We might want to make an issue going forward to use toIntExact instead of intValue() because I can see us getting overflows.